src/features/surveys/components/QuestionBlocks/QuestionBlockWrapper/QuestionBlockWrapper.tsx

```diff
@@ -107,7 +110,7 @@ export default function QuestionBlockWrapper({
         </div>
 
         <div className="mb-2 flex w-full items-center justify-end gap-2 sm:mb-0 sm:w-auto">
-          <div className="mr-1 sm:hidden">
+          <div className="mr-1 sm:hidden" data-tooltip-id={`my-tooltip-${index}`} data-tooltip-content={`${type} QUESTION`}>
             <QuestionTypeIcons type={type} />
```

Comment:
Please put tooltip it in QuestionTypeIcons component. Additionally please provide content of tooltip from translations